### PR TITLE
Exception asserter handles PHP 7 throwables

### PR DESCRIPTION
**Pull request by jubianchi**

### CODE REVIEW

File: classes/asserters/exception.php [hunk 1]
```diff
@@ -160,4 +160,9 @@ protected function check($value, $method)
 
 		return $this;
 	}
+
+	private static function isThrowable($value)
```
Comment (shouze) [inline classes/asserters/exception.php]: :+1:

**Comment by shouze** [comment]
LGTM thx @jubianchi !

**Comment by Hywan** [comment]
Tell me when it's ready to merge. It's :+1: for me.

**Comment by jubianchi** [comment]
@Hywan good for me ;)

**Comment by Hywan** [comment]
Thanks! :+1:

**Pull request merged**